maven-plugins/javadoc-maven-plugin/src/main/java/io/helidon/build/javadoc/JavadocMojo.java

```diff
@@ -750,8 +768,12 @@ private Set<SourceRoot> sourceRootsFromSourceJar(MavenArtifact dep) {
 
     private Set<SourceRoot> sourceRootsFromProjectFiles(Path dir) {
         List<Path> nested = Lists.filter(workspace.values(), it -> !it.equals(dir) && it.startsWith(dir));
+        DirectoryScanner ds = new DirectoryScanner();
```

Comment:
The DirectoryScanner seems unused.

Reply:
yes, discovered the `MavenFilters` class that does what I needed.